src/module/hook-multi-plexer/types.ts

```diff
@@ -0,0 +1,14 @@
+import { Address, Hex } from 'viem'
+
+export type SigHookInit = {
+  sig: Hex
+  subHooks: Address[]
+}
+
+export enum HookType {
```

Comment:
in typescript, do these also correspond to indexes or are they strings?

Reply:
@kopy-kat zero based indexes unless you pass explicit values